src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurBatchUtilityServiceImpl.java

```diff
@@ -149,6 +151,14 @@ public String getFileContents(String fileName) {
         }
     }
 
+    @Override
+    public boolean isValidTravelerStatus(String status) {
```

Comment:
would it make more sense to rename this to isVallidTravelerForEmployeePDPStatus?

Reply:
Ok, I'll see if I can rename this to better reflect what it is checking for.